src/Runners/PHPUnit/Options.php

```diff
@@ -425,6 +430,17 @@ public static function fromConsoleInput(InputInterface $input, string $cwd): sel
             $filtered['whitelist'] = $options['whitelist'];
         }
 
+        if (is_string($options['repeat'])) {
+            if (! is_numeric($options['repeat'])) {
+                throw new InvalidArgumentException(sprintf(
+                    'Option --repeat should have a number value, "%s" given',
+                    $options['repeat']
+                ));
+            }
+
+            $filtered['repeat'] = $options['repeat'];
```

Comment:
```suggestion
            if ($options['repeat'] !== (string) (int) $options['repeat']) {
                throw new InvalidArgumentException(sprintf(
                    'Option --repeat should have an integer number value, "%s" given',
                    $options['repeat']
                ));
            }

            $filtered['repeat'] = (int) $options['repeat'];
```

Reply:
`$filtered['repeat'] = (int) $options['repeat'];`

will request a chage on $filtered signature:

`* @param array<string, string|int|null> $filtered`

Okay on this?

Reply:
Sorry, didn't check that: ok for `$filtered['repeat'] = $options['repeat'];` then.